packages/theming/atlas/src/themesource/atlas_ui_resources/web/core/helpers/_badge.scss

```diff
@@ -6,43 +6,29 @@
 //
```

Comment:
Moving this file also means breaking change, right?

Reply:
No, I don't think so. All overriding label styles for Bootstrap are still present (`widgets/_labels.scss`). Devs using these label classes, like `label-success` on other elements/widgets will still get the correct styling. Besides that, devs should not touch these files, right?